.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "polkadot"]
+	path = polkadot
+	url = https://github.com/litentry/polkadot
```

Comment:
why not using upstream?

Reply:
We will try to set up test first with a local proved polkadot version (chain spec, Cargo.lock etc) first and once our test runs stably, It's good idea to migrate this dependency to upstream.